packages/qtwebengine.rb

```diff
@@ -51,75 +48,71 @@ class Qtwebengine < Package
-  depends_on 're2' => :build
+  depends_on 're2' # R
   depends_on 'snappy' # R
   depends_on 'zlibpkg' # R
 
   def self.patch
-    puts "@qt_pkgver is #{@qt_pkgver} and @qt_basever is #{@qt_basever}".lightblue
-    # Build & patches track Arch PKGBUILD at https://github.com/archlinux/svntogit-packages/raw/packages/qt5-webengine/trunk/PKGBUILD
-    Dir.chdir('src/3rdparty') do
-      system 'git checkout 87-based'
-      downloader 'https://github.com/archlinux/svntogit-packages/raw/packages/qt5-webengine/trunk/qt5-webengine-chromium-python3.patch',
-                 'fda4ff16790799fb285847918a677f4f3f7c0f513d4751f846ffc5aa5d873932'
-      system 'patch -Np1 -i qt5-webengine-chromium-python3.patch'
-      system "sed -i 's,#!/usr/bin/python,#!#{CREW_PREFIX}/python,g' chromium/build/print_python_deps.py"
-      downloader 'https://github.com/archlinux/svntogit-packages/raw/packages/qt5-webengine/trunk/qt5-webengine-ffmpeg5.patch',
-                 'c50d3019626183e753c53a997dc8a55938847543aa3178d4c51f377be741c693'
-      system 'patch -Np1 -i qt5-webengine-ffmpeg5.patch'
-      downloader 'https://github.com/archlinux/svntogit-packages/raw/packages/qt5-webengine/trunk/qt5-webengine-gcc12.patch',
-                 'cf9be3ffcc3b3cd9450b1ff13535ff7d76284f73173412d097a6ab487463a379'
-      system 'patch -Np1 -i qt5-webengine-gcc12.patch'
-      downloader 'https://github.com/archlinux/svntogit-packages/raw/packages/qt5-webengine/trunk/qt5-webengine-pipewire-0.3.patch',
-                 '5e3a3c4711d964d5152a04059a2b5c1d14bb13dd29bce370120f60e85b476b6f'
-      system 'patch -Np1 -i qt5-webengine-pipewire-0.3.patch'
-    end
-    downloader 'https://github.com/archlinux/svntogit-packages/raw/packages/qt5-webengine/trunk/qt5-webengine-python3.patch',
-               '398c996cb5b606695ac93645143df39e23fa67e768b09e0da6dbd37342a43f32'
-    system 'patch -Np1 -i qt5-webengine-python3.patch'
-    Dir.chdir('src/3rdparty/chromium/third_party') do
-      FileUtils.rm_rf 'catapult'
-      system 'git clone https://chromium.googlesource.com/catapult'
-      Dir.chdir('catapult') do
-        system 'git checkout 5eedfe23148a234211ba477f76fc2ea2e8529189'
-      end
-    end
-    # From https://www.linuxfromscratch.org/blfs/view/svn/x/qtwebengine.html
+    # Patches from lfs: https://www.linuxfromscratch.org/blfs/view/svn/x/qtwebengine.html
+    downloader 'https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.14-build_fixes-1.patch', 'SKIP'
+    system 'patch -Np1 -i qtwebengine-5.15.14-build_fixes-1.patch'
+    downloader 'https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.14-ffmpeg5_fixes-1.patch', 'SKIP'
+    system 'patch -Np1 -i qtwebengine-5.15.14-ffmpeg5_fixes-1.patch'
+    downloader 'https://www.linuxfromscratch.org/patches/blfs/svn/qtwebengine-5.15.14-icu_73-1.patch', 'SKIP'
+    system 'patch -Np1 -i qtwebengine-5.15.14-icu_73-1.patch'
+    FileUtils.mkdir_p '.git'
+    FileUtils.mkdir_p 'src/3rdparty/chromium/.git'
```

Comment:
These lines are unnecessary for a git repo build, right?  In any event, this is very hackish.

Reply:
As per https://www.linuxfromscratch.org/blfs/view/svn/x/qtwebengine.html These steps are specifically to avoid the source trying to update these subrepos...